front/lib/models/assistant/agent.ts

```diff
@@ -59,23 +53,6 @@ AgentGenerationConfiguration.init(
       allowNull: false,
       defaultValue: DataTypes.NOW,
     },
-    prompt: {
```

Comment:
Let make super duper 100% archi totally sure we have this in instructions for all models :grimacing:

Reply:
As there is ~no coming back from this one

Reply:
Other than firing a snapshot :/